### DISCUSSION TITLE
Survey of and Outreach to Parallel Communities

### DISCUSSION THREAD

**Discussion by aiaa-deic** [aiaa-deic/computation-ai#2] [General]
Collecting here linkage to communities outside of AIAA DEIC from which we can draw experience (and reduce our own replication of effort).

Initial examples:

- NAFEMS [Technical Working Groups](https://www.nafems.org/community/working-groups/) (including [HPC](https://www.nafems.org/community/working-groups/hpc/), [Multiphysics](https://www.nafems.org/community/working-groups/multiphysics/)) and [ASSESS](https://www.nafems.org/community/assess/) (Analysis, Simulation & Systems Engineering Software Strategies) **STEVE ARNOLD**
- NIST [MBE Program](https://www.nist.gov/programs-projects/model-based-enterprise-program), [Applied AI](https://www.nist.gov/applied-ai)
- HPCMP / CREATE-AV
### AIAA 
- [Multidisciplinary Design Optimization](https://engage.aiaa.org/aerospace-design-and-structures/communities/community-home?CommunityKey=aa1877c5-001b-4e1d-8b34-5e21ddfde04d), 
- [Design Engineering Technical Committee](https://engage.aiaa.org/aerospace-design-and-structures/communities/community-home?CommunityKey=43ad196b-14bd-4b34-938e-7958d0d5a381), 
- [Systems Engineering](https://engage.aiaa.org/aerospace-design-and-structures/communities/community-home?CommunityKey=bc3b07f8-3275-4cba-98cf-f7bd3fd7f874), 
- [CFD Vision 2030](https://engage.aiaa.org/aiaa-iod/communities/community-home?CommunityKey=2df568b8-b5e7-4f45-85eb-eda38e0a4c06) + [CFD 2030 site](https://www.cfd2030.com/index.html),  
- [Digital Engineering](https://engage.aiaa.org/aiaa-iod/communities/community-home?CommunityKey=a07dc652-85f5-4701-b09f-bbcdb62e533e) - see [Discussion topic: DEIC interdependencies](https://github.com/aiaa-deic/computation-ai/discussions/3)
- [Meshing, Viz, and Computational Environments](https://engage.aiaa.org/aerospace-sciences/communities/community-home?CommunityKey=e622185f-7808-4aeb-a341-efe3a48ad89a), 
- [Modeling and Simulation](https://engage.aiaa.org/aerospace-sciences/communities/community-home?CommunityKey=2207fdb2-0946-4f2d-8328-3e8d49e6aaa6), 
- [Multidisciplinary Design Optimization](https://engage.aiaa.org/aerospace-design-and-structures/communities/community-home?CommunityKey=aa1877c5-001b-4e1d-8b34-5e21ddfde04d)
- ICME Working Group - link? **STEVE ARNOLD**

**Others**? (Please add with links)

Further discussions - please suggest opportunities for sharing and aligning efforts.

**Comment by Papaghena** [comment]
Hi all, I am H Alicia Kim, professor, UC San Diego.  My research is in optimization for coupled problems, MDO, computational mechanics, topology optimization, material optimization.

I'm on the digital twin subcommittee and working with the digital workforce development efforts with DEIC and on ICME 2040 working group, MDO TC so I can be a bridge there.  I also have an officer position with the International Society for Structural and Multidisciplinary Optimization and US Association of Computational Mechanics and I can liaise with them too if and when needed.

**Comment by arthurrge** [comment]
Terry Hill (NASA HQ) presented on Digital Transformation at SciTech2024, including reference to this NASA/TM-20220018538:
[NASA's Digital Transformation Strategic Framework & Implementation Approach](https://ntrs.nasa.gov/api/citations/20220018538/downloads/2022-1206%20NASA%20TM%20-%20DT%20Strategic%20Framework%20%2B%20Implementation%20Plan%20(Marlowe%2C%20Haymes).pdf)

Great content there!

Note part of our discussion included a strategy for shaping the scope of this group within the (**excellent**!) framework developed in the (Steve Arnold-led) [NASA 2040 Vision](https://ntrs.nasa.gov/citations/20180002010) (_which is perhaps the seed of ICME 2040 as Steve and Alicia reference?_) -- e.g., see the "Key Element" list and interconnectedness.

**Comment by Papaghena** [comment]
Hi all,
Indeed, this document was the one we had been referring to.  I think it has more or less all the key components for digital engineering for material-structural systems (multiscale), and which have many commonalities with most complex systems.
Alicia

H Alicia Kim
Jacobs Scholar Chair Professor
Structural Engineering Department
Materials Science and Engineering
University of California San Diego
Franklin Antonio Hall, 3408
3180 Voigt Dr., Mail code 0433
La Jolla, CA 92093, USA
***@***.***
http://m2do.ucsd.edu

**Comment by arthurrge** [comment]
See also some of the themes-forward: National Academies' [Charting a Path in a Shifting Technical and Geopolitical Landscape](https://nap.nationalacademies.org/read/26916) / [DOI 10.17226/26916](https://nap.nationalacademies.org/catalog/26916/charting-a-path-in-a-shifting-technical-and-geopolitical-landscape)

**Comment by arthurrge** [comment]
I have recently connected with Jill Marlow, NASA Digital Transformation Officer - with a few potential follow-on threads. I was extremely impressed with the NASA sketch for digital transformation Terry Hill presented at SciTech. (Based on NASA/TM-20220018538 "[NASA's Digital Transformation Strategic Framework & Implementation Approach](https://ntrs.nasa.gov/api/citations/20220018538/downloads/2022-1206%20NASA%20TM%20-%20DT%20Strategic%20Framework%20%2B%20Implementation%20Plan%20(Marlowe%2C%20Haymes).pdf)" /Dec-2022).

One that aligns with this effort is **AI Strategy** - with NASA contacts:
- [Ed McLarney](https://www.linkedin.com/in/edwardlmclarney/) (Lead – Transformation Integration & AI/ML – NASA HQ - Office of CIO)
- [Dave Salvagnini](https://www.linkedin.com/in/dave-salvagnini-b432b0a9/) (Chief Data Officer - NASA HQ)

I'd be delighted to hear thoughts from our NASA members on this!

**Comment by arthurrge** [comment]
Synergy with [CFD 2030](https://www.cfd2030.com/) is pretty striking, perhaps we should hold a joint discussion with them (same Q re: ICME2040)
- CFD2030 Grand Challenges
   - [Numerical Prediction of High Lift Aerodynamics](https://arc.aiaa.org/doi/10.2514/6.2021-0955) 
   - [Full-engine CFD](https://ntrs.nasa.gov/citations/20205008206) (high-fidelity/accuracy) 
   - [CFD-in-the-loop](https://ntrs.nasa.gov/api/citations/20205003238/downloads/CFD-2030-Space-Vehicle-Grand-Challenge.pdf) 
- CFD2030 Vision Elements (see [fact sheet](https://www.cfd2030.com/CFD-2030-Fact-Sheet-2022-final.pdf)) – almost nothing in comp/AI scope outside of this, except non-CFD (e.g., materials)
   - physics-based, predictive modeling
   - simulation errors and uncertainties
   - automation in all steps of the analysis process
   - future HPC architectures
   - seamless integration with multi-disciplinary analyses and optimizations

***POTENTIAL CHALLENGE PROBLEM CONTEXTS***
- Disruptive Hardware (memory, accelerator, quantum, hybrid-AI, …)
   - Accelerators (GPU, AI/ML, physics), Novel Memory, FPGA/ASIC, Quantum, …
   - Synergies: [NAFEMS HPC](https://www.nafems.org/community/working-groups/hpc/), [NIST Applied AI](https://www.nist.gov/applied-ai), [DOE ECP](https://www.exascaleproject.org/), [DARPA](https://www.darpa.mil/about-us/advancing-national-security-through-fundamental-research)
- Predictive Accuracy (beyond VVUQ: realism+fidelity)
   - Surrogates /reduced-order models, automated mesh gen/validation
   - [CFD2030](https://www.cfd2030.com/), [DOE PCMM](https://www.osti.gov/servlets/purl/1480395), [NIST MBE](https://github.com/usnistgov/DT4SM/tree/master/MBE-Capabilities-Assessment), [NAFEMS HPC](https://www.nafems.org/community/working-groups/hpc/),
[NIST Applied AI](https://www.nist.gov/applied-ai)
- Multi-X (-physics, -disciplinary, -scale, -fidelity, …)
   - Surrogates /reduced-order models, automated mesh gen/validation
   - [CFD2030](https://www.cfd2030.com/), [NAFEMS Multiphysics](https://www.nafems.org/community/working-groups/multiphysics/), [DAF DTO](https://guide.dafdto.com/),
[NIST MBE](https://github.com/usnistgov/DT4SM/tree/master/MBE-Capabilities-Assessment), [AIAA MDO](https://engage.aiaa.org/aerospace-design-and-structures/communities/community-home?CommunityKey=aa1877c5-001b-4e1d-8b34-5e21ddfde04d), [AIAA Mod/Sim](https://engage.aiaa.org/aerospace-sciences/communities/community-home?CommunityKey=2207fdb2-0946-4f2d-8328-3e8d49e6aaa6)
- Enterprise Integration (ERP, PLM, MES, MRO, etc.)
   - Automation/standards ([SysML](https://sysml.org/), [FMI](https://fmi-standard.org/) )
   - AIAA DEIC [DECO](https://github.com/aiaa-deic/deco) (Digital Engineering Ecosystem)
- Data Mode Integration (tabular, time-series, geometry/tolerances, visual, unstructured, FAIR, machine-readability, training sets, …)
   - Automation / LLMs
   - [DAF DTO](https://guide.dafdto.com/), [NIST MBE](https://github.com/usnistgov/DT4SM/tree/master/MBE-Capabilities-Assessment), [GO-FAIR](https://www.go-fair.org/), [NIST Applied AI](https://www.nist.gov/applied-ai)
- Model VVUQ
   - Automated V&V, [maturity model](https://richardarthur.medium.com/co-design-web-6f37664ac1e1)
   - [ASME VVUQ](https://event.asme.org/VandV), AIAA DEIC “Model Trust”, [CFD2030](https://www.cfd2030.com/),
[DOE PCMM](https://www.osti.gov/servlets/purl/1480395)
- Model Maturity
   - Add scalability, maintainability, productivity / [maturity model](https://richardarthur.medium.com/co-design-web-6f37664ac1e1)
   - [DOE PCMM](https://www.osti.gov/servlets/purl/1480395), [NIST MBE](https://github.com/usnistgov/DT4SM/tree/master/MBE-Capabilities-Assessment), [NAFEMS Maturity](https://www.nafems.org/downloads/dropbox/resource_center/bm_jan_15_01.pdf), [INCOSE MBCM](https://www.incose.org/docs/default-source/default-document-library/leading-mbse-transformation_v5.pdf?sfvrsn=48e59bc6_0), [NASA SCL](https://ntrs.nasa.gov/api/citations/20120006603/downloads/20120006603.pdf)
- Model Competency / Bounding Definition (& lifecycle management)
   - Automated provenance capture, [maturity model](https://richardarthur.medium.com/co-design-web-6f37664ac1e1)
   - [NAFEMS ASSESS ESMS](https://www.nafems.org/publications/resource_center/assess_esms/), [NASA SCL](https://ntrs.nasa.gov/api/citations/20120006603/downloads/20120006603.pdf), [DOE PCMM](https://www.osti.gov/servlets/purl/1480395)
- Workforce / Culture (e.g., Asserting end-use confidence requirements for models)
   - [Modeling Literacy](https://drive.google.com/file/d/1DhkTct_4D5AElMoDeSOXsmBUISNtxdu1/view?usp=sharing) for decision-makers
   - [DAF DTO](https://guide.dafdto.com/), [NIST MBE](https://github.com/usnistgov/DT4SM/tree/master/MBE-Capabilities-Assessment)
- Human-machine Collaboration (sophisticated automation, collective intelligent stakeholder agents, generative design, …)
   - (including concepts outlined here: [Digital = Magic](https://richardarthur.medium.com/digital-magic-a702b2468ab6), [Provenance for Decision-Making](https://richardarthur.medium.com/provenance-for-decision-making-bf2c89d76ec2), [Machine-augmented Mindfulness](https://richardarthur.medium.com/machine-augmented-mindfulness-e844f9c54985))
   - [NIST Applied AI](https://www.nist.gov/applied-ai), [DAF DTO](https://guide.dafdto.com/), [DARPA](https://www.darpa.mil/about-us/advancing-national-security-through-fundamental-research)

**Reply by arthurrge** [reply]
Do we have anyone on the team who has been active with the CFD2030 group who might help coordinate a dialogue?

**Comment by arthurrge** [comment]
Re: DECO (Digital Ecosystem) - note they have a working paper:
[AIAA-DEIC-DigitalEcosystem-9Apr2024.docx](https://github.com/aiaa-deic/deco/files/14905221/AIAA-DEIC-DigitalEcosystem-9Apr2024.docx)

**Comment by Papaghena** [comment]
Hi,
The MDO TC has been involved with the CFD2030 group so unless someone in our group is directly involved, I am happy to facilitate a dialogue and the connection.
Alicia

H Alicia Kim
Chair
AIAA Multidisciplinary Design Optimization Technical Committee

Jacobs Scholar Chair Professor
Department of Structural Engineering 
Materials Science and Engineering
University of California San Diego
Franklin Antonio Hall, 3408
3180 Voigt Dr., Mail code 0433
La Jolla, CA 92093, USA
***@***.***
http://m2do.ucsd.edu

**Comment by arthurrge** [comment]
(XLS to be put in public versioning location: [Organizations and Technical Committees.xlsx](https://github.com/aiaa-deic/computation-ai/files/15277110/Organizations.and.Technical.Committees.xlsx) - has links to the groups)
Snapshot of tracking of relationships:
`<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/200002~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/200002~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">


Technical   Committee Names | Prioritized Committee   Description | CoDADE Member Will Contact
-- | -- | --
Adaptive   Structures | Autonomous structural systems   with sensors, processor, and actuators      to enable adaptation to changing   environmental conditions. |  
Aerodynamic   Measurement Technology | Advanced measurement techniques   for aerodynamic research in flight and ground-based facilities, including   surface and flow field measurements and visualization. |  
Aircraft   Design | Design and optimization of   complete aircraft systems, including analysis of their future potential. | Chris Saab
CFD 2030 Integration Committee | CFD Vision 2030 was established   to promote a community of practice engaged in developing methods, models,   physical experiments, software, and hardware for revolutionary advances in   computational simulation technologies for analysis, design, certification,   and qualification of aerospace systems.     CFD Vision 2030 serves as the focal point for activities to achieve the   NASA 2030 Vision by leveraging and integrating enabling technologies such as   high-performance computing, physical modeling,     numerical methods, geometry/grids, validation quality experiments, and MDAO   (multidisciplinary analysis and optimization) with quantified   uncertainties.     CFD Vision 2030 will communicate with other Committees to assure that the   AIAA membership engages with their peers and external constituencies in   shaping the future of simulation-based engineering. | Alicia Kim
Communication   Systems | Development, application, and   operation of communications systems     for point-to-point, mobile, aeronautical, maritime, and space users. |  
Computer   Systems | Theoretical and practical   considerations involving the applications     of computers and information processing techniques to aerospace programs. | Christopher Coley
Design   Engineering | Promotion of the development and   dissemination of design technologies     to assist the design engineer in creating and defining practical aerospace   products. | Ian Marks
Digital   Engineering Integration Committee | Digital Engineering Integration   Committee (DEIC) strives to accelerate integration of new and existing   ‘digital’ capabilities for improving US national competitiveness, security,   and operational readiness.  DEIC will   establish an Aerospace Forum for integrating currently disparate ‘digital’   activities     (e.g., Digital Twin, Digital Thread, ICME, data, etc).  DEIC will support constituent groups   (inside & outside AIAA) in the development & creation of a technical   programs related to Digital Engineering. | Marlon Rodgers, All of Us
Flight   Testing | Testing of aircraft, spacecraft,   and missiles in their natural environment with regard to research,   development, acceptance, and operational testing of these craft and/or   associated equipment. |  
Human-Machine   Teaming | Foster the community,   methodologies, and technologies that will enable safe, trusted, and effective   integration of humans and complex machines in aerospace and related domains. |  
Intelligent   Systems | Application of Intelligent   System technologies and methods to aerospace systems, the verification and   validation of these systems, and education in the use of IS technologies in   aerospace. |  
Materials | Systematic application of   materials to aerospace systems, and interdisciplinary approach to the   development, testing, and evaluation of different materials and associated   technologies. | Steve Arnold
Meshing,   Visualization,     and Computational Environments | Application of computer science   to pre- and post-processing, and infrastructure     in support of computational simulation in the Aerospace Community. |  
Modeling   and Simulation | Simulation of atmospheric and   space flight conditions in ground-based simulators and aircraft     for aircrew training, design, development, and evaluation of aerospace   systems. |  
Multi-Disciplinary   Design Optimization | Dissemination of the   development, application, and teaching of a design methodology integrating   disciplinary analyses, optimization, and AI in design of aerospace systems. | Vladimir Balabanov, Alicia Kim
Non-Deterministic   Approaches | Advancing the relevant art,   science and cross-cutting technologies required     for the successful application of non-deterministic approaches to aerospace   systems. |  
Product   Support | Advancing aircraft logistics and   support systems in engineering and supply support, reliability-based   logistics, aircraft maintenance, mod and repair, reliability, and aircraft   integrity programs. | Abhishek Ram
Sensor   Systems and Information Fusion | Sensor systems and distributed   sensor networks for detection, collection, information fusion,     storage, retrieval, distribution, and reception of information at various   levels. |  
Survivability | Promotion of survivability as a   design discipline, including both the survivability assessment methodology   and the survivability enhancement technology, for air and space systems. |  
Systems   Engineering | Fostering the definition,   dissemination, development, understanding and application of system   engineering processes, methodologies and tools to aerospace, computer, and   ground systems. |  
INCOSE AI   Systems | This   WG will address use of AI within Systems and using AI as part of the Systems   development. |  
NAFEMS+INCOSE   Systems Modeling & Simulation WG | The mission of the SMS WG is to   develop a vendor-neutral, end-user driven consortium that     not only promotes the advancement of the technology and practices   associated with integration     of engineering simulation and systems engineering, but also acts as an   advisory body     to drive strategic direction for technology development and international   standards     in the space of complex engineering. |  
INCOSE   DIGITAL END WG | The   Digital Engineering Working Group supports the strategic objective of     accelerating digital engineering adoption and the role of system engineers   in this new paradigm. | Chris Saab
NAFEMS ASSESS | The   NAFEMS ASSESS Initiative is a broad reaching multi-industry initiative with a   primary goal     to facilitate this revolution of enablement that will vastly increase the   availability and utility     of Engineering Simulation, leading to significantly increased business   benefits across the full spectrum     of industries, applications and users.     The ASSESS Initiative brings together key players from industry,   government, academia and     the software providers to guide and influence the software strategies for   performing     model-based analysis, simulation, and systems engineering. | Rick Arthur
NAFEMS HPC   Working Group | (Also WG's in CFD, Structural   Mechanics, Multiphysics, etc.) | Vladimir Balabanov, Alicia Kim
ADIC |   | Gregory Roth



</body>

</html>

**Reply by imvg2024** [reply]
Update from my side: I have talked with DETC. One challenge is that may is when some TCs change leadership, so transition may delay discussion. Will hopefully have more updates in the coming days with email traffic. 

- Ian

**Reply by aramdraper** [reply]
Hi Ian, thank you for the update!!

**Reply by ChadHall1** [reply]
Added contacts into attached spreadsheet for the below TCs:
- Structures Technical Committee (Aerospace Design & Structures Group)
- Guidane, Navigation and Control (GNC) TC and the newly formed International Aerospace Coalition Outreach Committee
- Propulsion and Energy Group (Inlets, Nozzles and Propulsion Systems Integration (INPSI) sub committee contact
[Organizations.and.Technical.Committees 002.xlsx](https://github.com/user-attachments/files/16057957/Organizations.and.Technical.Committees.002.xlsx)
)

**Comment by arthurrge** [comment]
FYI Abhishek and I just met with AIAA + McKinley regarding the survey, the rough timeline looks to be:
Finalization (23-Oct)
Launch (28-Oct)
Close (11-Nov)
Data analysis (22-Nov)

We will want to enlist everyone to use your networks to make sure this reaches and motivates people to respond!
McKinley noted roughly 10% response rate is expected and that we would want 400+ responses to have tolerable error in the data.

So - please start thinking about the aerospace/defense communities we wish to reach and gather!
